tasks.py

```diff
@@ -526,6 +532,9 @@ def vscode(context):
     """Launch Visual Studio Code with the appropriate Environment variables to run in a container."""
     command = "code nautobot.code-workspace"
 
+    if not HAS_PYYAML:
+        raise Exit("You must install pyyaml ('pip install --user pyyaml') to use this command.")
```

Comment:
Should we recommend `poetry install` instead of `pip install --user pyyaml`? Since pyyaml is part of our project dependencies.

Reply:
`pyyaml` is already in our poetry.lock. You can run any of our django projects with the docker dev environment with only `invoke` installed on the local machine. We usually install the poetry project so that our IDE can see the dependencies but you don't have to `poetry shell` to use any of the invoke tasks.

Reply:
Understood. I just wonder if having people locally install pyyaml is preferable to just using poetry shell?

Reply:
We could just change the error message to `PyYAML is required to run this command` and let the developer decide what to do?

Reply:
Probably better to have some guidance - I can accept the current text.